### PR TITLE
add day01 ex02

### PR DESCRIPTION
**Pull request by ktnltr**

### CODE REVIEW

**Review by 0xChqrles — CHANGES_REQUESTED** [review]
NORME

File: C01/ft_ft.c
Comment (0xChqrles) [inline C01/ft_ft.c:1]: Duplicated file, please remove

**Review by 0xChqrles — CHANGES_REQUESTED** [review]
fix comments

File: C01/ft_ft.c
Comment (ktnltr) [reply C01/ft_ft.c:1]: done

**Review by 0xChqrles — APPROVED** [review]
lgtm

**Pull request merged**